pact.openapi.yaml

```
@@ -549,12 +550,23 @@ components:
               example:
                 "1.23498218000001"
       - title: Integer
-        description: Pact integers use a special schema.
+        description: There are two alternative JSON representations for Integer.
+        type: object
         required: [int]
         properties:
           int:
-            type: number
-            description: JSON number to be truncated to an integer value.
+            oneOf:
+            - title: Number
+              description: |
+                JSON numbers can be used whenever the precision is adequate
```

```suggestion
                JSON numbers are rounded to integer values.
```
I _think_ this is "normal" rounding (`RealFrac` docs are really weird about this but Scientific looks like it does the "normal" thing? https://hackage.haskell.org/package/scientific-0.3.7.0/docs/src/Data.Scientific.html#line-564

---

Is this doing rounding?  I was thinking that since it's integer already no rounding would be needed.